test/skimNtuple_HHbtag.cpp

```diff
@@ -2786,21 +2788,20 @@ int main (int argc, char** argv)
 			}
 
 			else if (pType == 1) {
-				//TODO: should be super cluster eta (not available in bigntuples at the moment)
 				float leg1_eleReco_SF;
 				float leg1_eleReco_SFerr;
 				// ele RECO SFs are split into two categories: pt <= 20 and pt > 20
 				if(leg1pt <= 20){
-					leg1_eleReco_SF = lepSFs[2]->get_direct_ScaleFactor(leg1pt, leg1eta, pType);
-					leg1_eleReco_SFerr = lepSFs[2]->get_direct_ScaleFactorError(leg1pt, leg1eta, pType);
+					leg1_eleReco_SF = lepSFs[2]->get_direct_ScaleFactor(leg1pt, leg1eta_SC, pType);
```

Comment:
Just to confirm, is there no "super cluster pT" available/needed?

Reply:
AFAIK that doesn't exist